Tools/Audit-Sentinel-Detection/Readme.md

```diff
@@ -0,0 +1,80 @@
+
+
+# Audit Sentinel Detection Rules
+
+### Prerequistes : [Configure Audit in Microsoft Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/enable-monitoring)
+
+### Purpose : Detection / Analytics are dircetly changed in the production instace at times due to several reaons.
+
+ + As the DevOps workflow is not set up.
+ + Due to urgency.
+
+Here is KQL query to idetify such scenarios and an azure function developped in KQL which can be used to display the changes.
```

Comment:
@samikroy , fix these typos as well idetify to identify,  developped to developed

Reply:
Fixed